src/platform/plugins/private/links/public/plugin.ts

```diff
@@ -99,10 +101,14 @@ export class LinksPlugin
         stage: 'production',
         appExtensions: {
           visualizations: {
-            docTypes: [CONTENT_ID],
+            docTypes: [LINKS_SAVED_OBJECT_TYPE],
             searchFields: ['title^3'],
             client: getLinksClient,
-            toListItem(linkItem: LinksCrudTypes['Item']) {
+            toListItem(
```

Comment:
Type change from `LinksCrudTypes['Item']` needed because `LinksAttributes` in src/platform/plugins/private/links/common/content_management/v1/types.ts defined `links` as optional. This is not how `links` is defined in `linksSchema.` `toListItem` types the parameter as `VisualizationSavedObject`. The real issue is that `toListItem` does not type the parameter as a generic so there was a casting error when casting `VisualizationSavedObject` to `LinksCrudTypes['Item']` because `attributes.links` does not exist in  `VisualizationSavedObject`.